front-end/src/pages/Admin/AdminMaterials/MaterialForm/MaterialForm.jsx

```diff
@@ -123,8 +130,16 @@ const CreateMaterial = ({ material, levels, create }) => {
         setIsLoading(true);
         await createMaterial(materialData);
         setIsLoading(false);
+        snackbarShowMessage({
+          message: 'Material saved',
+          severity: 'success',
+        });
       } else {
         await updateMaterial(material._id, materialData);
+        snackbarShowMessage({
```

Comment:
You can place snackbarShowMessage after if statement to avoid duplication.

Reply:
done